test/e2e/framework/test_context.go

```diff
@@ -79,6 +79,7 @@ type TestContextType struct {
 	NodeTestContextType
 
 	// Viper-only parameters.  These will in time replace all flags.
+
```

Comment:
added this space b/c now there are more than one params, gives readable separation.